editor/src/templates/editor-entry-point.tsx

```diff
@@ -19,9 +19,11 @@ import { addStyleSheetToPage } from '../core/shared/dom-utils'
 import { STATIC_BASE_URL } from '../common/env-vars'
 
 const editorCSS = [
-  `${STATIC_BASE_URL}editor/css/canvas.css`,
-  `${STATIC_BASE_URL}editor/css/slider.css`,
-  `${STATIC_BASE_URL}editor/css/ReactContexify.css`,
+  `${STATIC_BASE_URL}editor/initial-load.css`,
+  `${STATIC_BASE_URL}editor/canvas.css`,
+  `${STATIC_BASE_URL}editor/slider.css`,
+  `${STATIC_BASE_URL}editor/cursors.css`,
```

Comment:
seems like someone deleted cursors.css from the editor-entry-point recently?

Reply:
@maltenuhn it was you.

Reply:
Ah yes. It was already empty.